providers/catalog.go

```diff
@@ -156,6 +157,25 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 		},
 	},
 
+	// Copper configuration
+	Copper: {
+		AuthType: Oauth2,
+		BaseURL:  "https://api.copper.com/developer_api/v1",
```

Comment:
@Listener430 can you please remove /v1 from the base URL? We don't want to have to update this if they release a v2.